tests/admin_views/tests.py

```diff
@@ -71,14 +71,10 @@ def get_admin_form_fields(self, response):
         admin_form = response.context['adminform']
         fieldsets = list(admin_form)
 
-        field_lines = []
-        for fieldset in fieldsets:
-            field_lines += list(fieldset)
-
         fields = []
-        for field_line in field_lines:
-            fields += list(field_line)
-
+        for fieldset in fieldsets:
+            for field_line in fieldset:
+                fields.extend(field_line)
```

Comment:
I guess we could `return list(itertools.chain.from_iterable(admin_form))` as well.

https://docs.python.org/3/library/itertools.html#itertools.chain.from_iterable

Reply:
Yes, but IMHO it's worse for readability.

Reply:
Was going to suggest the same, but see that @charettes got there first.

> Yes, but IMHO it's worse for readability.

Fair enough.